contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortSwapJob.java

```diff
@@ -121,15 +121,15 @@ public void run() throws Exception {
 
         // While processing an admin request, HDFSFailedLock could take long time because of multiple HDFS operations,
         // especially when the name node is in a different data center. So extend timeout to 5 minutes.
-        AdminClientConfig adminConfig = new AdminClientConfig().setMaxConnectionsPerNode(cluster.getNumberOfNodes())
+        AdminClientConfig adminConfig = new AdminClientConfig().setMaxConnectionsPerNode(cluster.getNumberOfNodes()) // TODO: Figure out why do we need this config?? -FGV
```

Comment:
Ok thanks. I've changed my comment to integrate the info you provide above. I'm leaving the setting as is.